### ISSUE TITLE
unsupportd type uint8 error for CropMirrorNormalize

### ISSUE THREAD

**Issue by Vert53** [NVIDIA/DALI#2453]
Getting the following error for CropMirrorNormalize which is supposed to support uint8 according to documentation.
[/opt/dali/dali/operators/image/crop/crop_mirror_normalize.cc:93] Not supported output type:uint8
I'm using version 0.27 of dali
Here's the code:
```
class CropMirrorNormalizePipelineFromNumpy(Pipeline):
    def __init__(self, batch_size, num_threads, device_id):
        super().__init__(batch_size, num_threads, device_id)
        self.input = ops.NumpyReader(file_root=image_dir, random_shuffle=False,
                                     file_list=join(image_dir, 'imgs_as_numpy/file_list.txt'))
        self.reshape = ops.Reshape(device="cpu", layout="HWC")

        std = [1 / (0.229 * 255),
               1 / (0.224 * 255),
               1 / (0.225 * 255)]
        mean = [1 / (0.485 * 255),
                1 / (0.456 * 255),
                1 / (0.406 * 255)]
        self.crop_mirror_normalize_0 = ops.CropMirrorNormalize(device="cpu",
                                                               output_layout='HWC',
                                                               std=std,
                                                               mean=mean,
                                                               crop_w=224,
                                                               crop_h=224,
                                                               crop_pos_z=0,
                                                               dtype=DALIDataType.UINT8
                                                               )

    def define_graph(self):
        images = self.input()
        images = self.reshape(images)
        output_images = self.crop_mirror_normalize_0(images)
        return output_images
```

**Comment by JanuszL** [comment]
Hi,
It is a mistake in our documentation, https://github.com/NVIDIA/DALI/pull/2454 should fix that. You can set the output type to Float and then use the Cast operator to downcast to uint8.

**Comment by JanuszL** [comment]
https://github.com/NVIDIA/DALI/pull/2458 has been merged, and support for UINT8/INT8 has been added. It will be available in the next nightly build and in 0.29.

**Comment by JanuszL** [comment]
0.29 has been released and should address this issue.
